### DISCUSSION TITLE
SNAPSHOT-20210703.151310

### DISCUSSION THREAD

**Discussion by qvacua** [qvacua/vimr#853] [General]
⚠️ Please note:

* This release may be instable!
* This is *not* a universal binary. Universal binary will be released again as soon as @apple releases those shiny new MBPs 😉

---

## Since the last snapshot:

* Neovim 0.5.0 😄 
* GH-750: Search highlight displays incorrectly for gui=reverse: Thanks @rluba for the PR!
* Dependencies updates:
    - PureLayout/PureLayout@3.1.9
    - iwasrobbed/Down@0.11.0
    - sparkle-project/Sparkle@1.26.0
    - Quick/Nimble@9.2.0
    - ReactiveX/RxSwift@6.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/qvacua/vimr/releases/tag/snapshot/20210703.151310'>SNAPSHOT-20210703.151310</a>.</em>

**Comment by wellcomez** [comment]
You are last hope for neovim gui on earth.
 problem1.   just on buffer tab can be displayed .

**Reply by wellcomez** [reply]
<img width="819" alt="截屏2021-09-18 上午9 41 09" src="https://user-images.githubusercontent.com/2755665/133868054-470ffbe8-32e9-4fb5-a759-96bf772a2316.png">

**Comment by finviman** [comment]
Apple will release new MBP next Monday.

**Reply by qvacua** [reply]
Let's hope... Then, I could build a universal version of the app.

**Reply by wellcomez** [reply]
It’s my problem. Not a bug. After turn off vimr tab in preference  ，nvim’s default tabline appears . My concern is that no everybody knows such problem is caused by an optional setting of vimr.
